src/App.js

```diff
@@ -0,0 +1,74 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { AiFillPlusCircle } from "react-icons/ai";
+import TaskList from "./components/ui/TaskList";
+const App = () => {
+  const [task, setTask] = useState("");
+  const [toDo, setAllToDo] = useState([]);
+
+  const handleOnChange = (event) => {
+    setTask(event.target.value);
+  };
+  const handleTasksubmit = (event) => {
+    if (task !== "") {
+      const newTask = { id: nanoid(), taskName: task, isTaskDone: false };
+      event.preventDefault();
```

Comment:
I don't really get why you used this event.preventDefault method while you did not use the form. I thought it is not needed,